### PR TITLE
[5.1] Use Dialog for Batch windows

### PR DESCRIPTION
**Pull request by Fedik**
### Summary of Changes

Use Dialog for Batch windows
The same as https://github.com/joomla/joomla-cms/pull/40359 but for rest of Batch buttons.

### Testing Instructions

Run `npm install`
Test Batch dialog for:
- banners
- contacts
- newsfeeds
- fields
- field groups
- menus
- modules
- tags
- users

### Actual result BEFORE applying this Pull Request
Works

### Expected result AFTER applying this Pull Request
Works

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/184
- [ ] No documentation changes for manual.joomla.org needed

References:
- https://github.com/joomla/joomla-cms/pull/40359
- https://github.com/joomla/joomla-cms/pull/40150

### CODE REVIEW

File: build/media_source/com_fields/js/admin-fields-default-batch.es6.js [hunk 1]
```diff
@@ -2,18 +2,22 @@
...
   batchSelector.addEventListener('change', () => {
     if (batchSelector.value === 'nogroup' || batchSelector.value !== '') {
       batchCopyMove.classList.remove('hidden');
     } else {
       batchCopyMove.classList.add('hidden');
     }
   }, false);
-});
+};
+document.addEventListener('DOMContentLoaded', () => setUp(document));
```
Comment (dgrammatiko) [inline build/media_source/com_fields/js/admin-fields-default-batch.es6.js]: If it's `defer` or `type=module` then
```suggestion
setUp(document);
```

Comment (Fedik) [reply build/media_source/com_fields/js/admin-fields-default-batch.es6.js]: It does not make diference, but I will allow it here :)

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 6b7061f97dcd5ce06cf3dbc3756b25f3f8ed8047<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42328">issues.joomla.org/tracker/joomla-cms/42328</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 6b7061f97dcd5ce06cf3dbc3756b25f3f8ed8047<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42328">issues.joomla.org/tracker/joomla-cms/42328</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42328">issues.joomla.org/tracker/joomla-cms/42328</a>.</sub>

**Comment by LadySolveig** [comment]
Thank you for your work @Fedik !
And of course many thanks to our testers and @viocassel @Quy and @dgrammatiko for the review.

**Pull request merged**